internal/server/data/settings.go

```diff
@@ -62,9 +63,12 @@ func initializeSettings(db *gorm.DB) (*models.Settings, error) {
 
 func GetSettings(db *gorm.DB) (*models.Settings, error) {
 	org := OrgFromContext(db.Statement.Context)
+	return getSettingsForOrg(db, org.ID)
+}
 
+func getSettingsForOrg(db *gorm.DB, orgID uid.ID) (*models.Settings, error) {
 	var settings models.Settings
-	if err := db.Where("organization_id = ?", org.ID).First(&settings).Error; err != nil {
+	if err := db.Where("organization_id = ?", orgID).First(&settings).Error; err != nil {
```

Comment:
Are we moving away from using the `data.go` `get()` function? Trying to figure out why this is doing the call directly.

Reply:
Pretty soon we will be for sure, with the removal of gorm.  This I cherry-picked from `org_collab`, so I'm not sure exactly of the reason for doing it this way instead of the selector.  

Either way, I'll be replacing all of the SelectorFuncs and gorm stuff with raw sql, which will look more like this.